### PR TITLE
Whitelabel Tile Server

### LINKED ISSUE #510: Configure Tile Server for each Project
**Is your feature request related to a problem? Please describe.**
Currently we are using martin as a tile server. Because we will have now multiple projects we need different endpoints for each project.

**Describe the solution you'd like**
* The tile layer (physical_stores)(https://tiles.ehrenamtskarte.app/rpc/physical_stores.json) should be copied. The new copy should have the name `bayern.ehrenamtskarte.app-physical_stores`.
* A new tile layer `nuernberg.sozialpass-physical_stores` needs to be created

For each layer we need to define an SQL query which queries the stores based on their project ID. This maybe required to change the definition of several pgsql functions.

**Describe alternatives you've considered**
Maybe we want to switch to the t-rex tile server with this task (#458). **Please check which tile server offers easier configuration**

**Additional context**
None

### PR DESCRIPTION
**Pull request by maxammann**
Fixes #510

### CODE REVIEW

**Review by michael-markl — APPROVED** [review]
Looking great!

File: administration/.env.development [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_API_BASE_URL=http://localhost:7000
+REACT_APP_API_BASE_URL=http://localhost:8000
```
[the remark this answers is not in the document]
Comment (maxammann) [reply administration/.env.development:1]: macOS is blocking prot 7000... So I thought it is better to switch.

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/map/webservice/MapStyleHandler.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    fun handle(context: Context) {
+        try {
+            val projectId: String = context.pathParam("project_id")
+            val style: String = this.styles[projectId]!!
+            context.result(style)
+            context.contentType("application/json")
+        } catch (e: IOException) {
+            context.res().sendError(500)
+        } catch (e: Throwable) {
+            println(e)
```
[the remark this answers is not in the document]
Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/map/webservice/MapStyleHandler.kt:87]: Oh that is a left-over from debuggin, good catch! I think we actually do not need to catch most exceptions. We could use: https://javalin.io/documentation#exception-mapping

Somehow we should find a pattern for stuff like that.

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/pipelines/SozialpassNuernberg.kt [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+                PhysicalStoreEntity.new {
+                    storeId = storeEntity.id
+                    addressId = address.id
+                    coordinates = Point(11.061859, 49.460983)
+                }
+            } catch (e: Exception) {
+                logger.info("Unknown exception while storing to db", e)
+                rollback()
+                throw e
+            }
```
[the remark this answers is not in the document]
Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/pipelines/SozialpassNuernberg.kt:52]: I copy pasted this from the bayern pipeline, actually no if using nested transactions: https://github.com/JetBrains/Exposed/wiki/Transactions#using-nested-transactions

**Pull request merged**